test/src/tests.js

```diff
@@ -189,6 +189,121 @@ describe('Rokt Forwarder', () => {
         });
     });
 
+    describe('#hashAttributes', () => {
+        beforeEach(() => {
+            window.Rokt = new MockRoktForwarder();
+            window.mParticle.Rokt = window.Rokt;
+            window.mParticle.Rokt.attachKitCalled = false;
+            window.mParticle.Rokt.attachKit = async (kit) => {
+                window.mParticle.Rokt.attachKitCalled = true;
+                window.mParticle.Rokt.kit = kit;
+                Promise.resolve();
+            };
+            window.mParticle.forwarder.launcher = {
+                hashAttributes: function (attributes) {
+                    window.mParticle.Rokt.hashAttributesOptions = attributes;
```

Comment:
nit - They aren't really options.  Why not just assign to attributes?
```suggestion
                    window.mParticle.Rokt.attributes = attributes;
```

Reply:
Technically hashed attributes and attributes (placement attributes) are different and the `hashAttributes` _shouldn't_ touch the placement attributes.

Reply:
gotcha, yep I like `hashedAttributes` then like you added